Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -84,7 +84,11 @@ protected override void Awake()
             }
             else
             {
-                XRDevice.SetTrackingSpaceType(transparentTrackingSpaceType);
+                // Removed for now, until the HoloLens tracking space type story is more clear.
+                //XRDevice.SetTrackingSpaceType(transparentTrackingSpaceType);
+
+                enabled = false;
+                return;
```

Comment:
Bounds isn't nullable, so it won't nullref
![image](https://user-images.githubusercontent.com/3580640/34012584-3620f278-e0c9-11e7-90ff-e8dec7b44067.png)

Addressing Stephen's comment below about destroying instead of disabling will prevent the method from being called as well.